games-action/minetest/metadata.xml

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<herd>games</herd>
```

Comment:
> but i hope that this issue gets fixed soon, otherwise i don't see how reasonable work is possible

We are working on it.

Anyway, since you are proxy-maintainer, I am happy to merge this if you keep the `metadata.xml` as-is. Also mind that all of this does not apply if you import a new ebuild. Then you are not obliged to add the gamers herd.